src/test/java/step1/StringTest.java

```diff
@@ -0,0 +1,66 @@
+package step1;
+
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.*;
+
+public class StringTest {
+
+    @Test
+    @DisplayName("1,2을 ,로 split 했을 때 1과 2로 분리된다.")
+    void split1() {
+        String text = "1,2";
+
+        String[] actual = text.split(",");
+        String[] expected = {"1", "2"};
+
+        assertThat(actual).isEqualTo(expected);
+    }
+
+    @Test
+    @DisplayName("1을 ,로 split 했을 때 1만을 포함하는 배열이 반환된다.")
+    void split2() {
+        String str = "1";
+
+        String[] actual = str.split(",");
+        String[] expected = {"1"};
+
+        assertThat(actual).isEqualTo(expected);
```

Comment:
```suggestion
        assertThat("1".split(",")).contains("1");
```
극단적으로 위와 같이 한 라인으로 구현해 보는 것은 어떨까?